QuowMinimap.lua

```diff
@@ -13759,7 +13762,7 @@ function QuowHandleScoreBrief(sName, sLine, wildcards, styles)
   OnPluginMXPsetEntity("hp="..wildcards[1], true)
   OnPluginMXPsetEntity("gp="..wildcards[3], true)
   -- Also ignore xp if no burden to avoid broken combat monitors
-  if (wildcards[6] ~= "") then
+  if (wildcards[6] ~= "" and wildcards[6] ~= nil) then
```

Comment:
The `wildcards` array that SmushClient passes to trigger callbacks is a long as the number of wildcards matched. That means if only five wildcards match, `wildcards[6]` will be `nil` rather than an empty string.